docs/src/introduction/overview.md

```diff
@@ -134,139 +170,62 @@ fuel-core --db-type in-memory
 
 This starts a Fuel node with a volatile database that will be cleared when shut down (good for testing purposes).
 
-### Deploy `wallet_contract` To Your Local Fuel Node
+### Deploy `counter_contract` To Your Local Fuel Node
 
-To deploy `wallet_contract` on your local Fuel node, run
+To deploy `counter_contract` on your local Fuel node, run the following command back in your original terminal so you don't end the process running the local Fuel node, from the root of the `wallet_contract` directory:
```

Comment:
I find this paragraph a little confusing, maybe we could say something like "note that you cannot use the same terminal session for both fuel-core and running other commands like `forc`"?